src/Index.js

```diff
@@ -901,6 +942,18 @@ Index.prototype.batchRules = function(rules, opts, callback) {
   });
 };
 
+Index.prototype.replaceAllRules = function(rules, opts, callback) {
```

Comment:
We probably should discuss if we should or not verify the `objectID` of the provided rules.

Reply:
I don't think we should, engine will return an error if no objectID is set.